client/lib/site/index.js

```diff
@@ -86,46 +87,7 @@ Site.prototype.set = function( attributes ) {
  * Update any computed attributes
  */
 Site.prototype.updateComputedAttributes = function() {
-	/**
```

Comment:
Still think this is yet to be addressed https://github.com/Automattic/wp-calypso/pull/6477/files#r73341185